docs/source/_sidebar.yaml

```diff
@@ -1,9 +1,16 @@
+banner:
+  message: "You are viewing documentation for a preview version of this software."
+  link:
+    href: "./#previews"
+    content: "Learn about previews."
 switcher:
   heading: "Apollo Kotlin"
   versions:
+    - label: v5
+      href: ./
     - label: v4
       latest: true
-      href: ./
+      href: ./v4
     - label: v3
       href: ./v3
     - label: v2
```

Comment:
All other hrefs need to be `../` when the branch isn't on `latest`

See v3: https://github.com/apollographql/docs-rewrite/blob/main/content/pages/(developer-tools)/kotlin/v3/_sidebar.yaml

Reply:
https://github.com/apollographql/apollo-kotlin/pull/6504/commits/7d0e6f3085441a26f90abc6cf7ebcb756857b3f9